config/initializers/figaro.rb

```diff
@@ -1,9 +1,11 @@
 # config/initializers/figaro.rb
 
-Figaro.require_keys("S3_KEY",
-                    "S3_SECRET",
-                    "S3_BUCKET",
-                    "S3_REGION",
-                    "EMAIL_USERNAME",
-                    "EMAIL_PASSWORD"
-                    )
+Figaro.require_keys(
+  "EMAIL_USERNAME",
+  "EMAIL_PASSWORD",
```

Comment:
Lol this is sort of my fault, but we don't need EMAIL_USERNAME and EMAIL_PASSWORD when you're on local (not on Heroku, cause we use SendGrid).  Is there a way to configure this for the environment?